### DISCUSSION TITLE
How to use the repository / GH features

### DISCUSSION THREAD

**Discussion by markusschloesser** [markusschloesser/MackieC4_P3#50] [General]
if we open a new issues:
1. assign one project member
2. assign the issue to the project, that way progress is tracked automatically in the kanban board

**Comment by BilldarBagdar** [comment]
is "automatically tracking progress on the kanban board" a feature we need or want?  I understand the utility of a "kanban board" for example, but I don't necessarily see how "tracking progress" adds any value over and above the value we already derive from "Issues".

"Progress tracking" is a distinctly managerial need/desire, IMHO. :)   I mean if I'm waiting for you to get some work done, I can just ask you when you think you will have whatever ready.  What value do I gain by going instead to the "kanban board" and checking your reported progress on "the story/Issue"?  I'm generally fairly allergic to "bureaucratic paperwork"...  :)

**Reply by markusschloesser** [reply]
I thought about it and tried it out. Opening a new issue automatically assigns that to the todo cat, which I like, cos less paperwork to do ;-) etc
I also really like the board! Great idea! :-)
I think it will help me keep better track of things, I already found stuff today 😁

**Comment by BilldarBagdar** [comment]
Yes, I agree.   I just like, you know, um, noticed that Projects Tab...  :)  Is it new?  I don't think that GitHub feature is enabled in the version I use at work.

**Reply by markusschloesser** [reply]
I did that yesterday 😊

**Comment by markusschloesser** [comment]
Copied from issue thread where it was OT:

> Even more threads like "today's repository status" that are more general places to talk about various things. Like instead of a new thread to say check out the fix in branch xxx, it's just a new post in a "standard thread" for that kind of stuff. If we create an issue, we've already discussed what the issue is and don't need to have the same discussion "in the issue thread". I'm not saying that because you just posted a new issue, not respecting "frozen" is definitely a bug. But in general, a sort of pre-issue discussion thread where any possible issue discussion first appears might be useful. Another possible such thread would be for the #MS comments / questions we want to discuss. If I want to return-comment or answer your questions or vice versa, we have a common thread for that kind of thing. We can just cut/paste the comments from PyCharm to browser and comment further or whatever.

I like using issues, especially now that they are automatically tracked and moved in the project board. So I'm against pre-discussing things 😊. IMHO it would just multiply managerial tasks, which I don't wanna do. And as you said, it's our repository, so we've got nothing to hide. Plus we can always move issues to the "not applicable" column or delete them
